### DISCUSSION TITLE
why does `expo start --android` requires a development build?

### DISCUSSION THREAD

**Discussion by dchhetri** [expo/expo#26086] [Archived: Questions]
If using the starter typescript template, I'm not seeing it require a development build. But if I try that in my custom project -built on top of the typescript template project, it fails with the following error
```
npx expo start --android
CommandError: No development build (com.loqal.my.project) for this project is installed. Please make and install a development build on the device first.
```

anyone have any idea why it's requiring a development build and just not launching expo in the android emulator?

**Comment by andrconstruction** [comment]
Just remove expo-dev-client

**Reply by jaysonsosweet** [reply]
Thx. Solved the problem for me.

**Comment by KanoDekouBillyBrown** [comment]
@dchhetri Did removing the expo-dev-client solve your problem , I'm currently facing the same though i built mine and used orbit to open it in the emulator
